README.md

```diff
@@ -21,6 +21,24 @@ To easily install or upgrade to the latest release, use [pip](http://www.pip-ins
 pip install --upgrade ShopifyAPI
 ```
 
+### Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Public and Custom Apps](#public-and-custom-apps)
+  - [Private Apps](#private-apps)
+- [Billing](#billing)
+- [Session Tokens](docs/session-tokens)
+- [Handling Access Scope Operations](docs/api-access.md)
+- [Advanced Usage](#advanced-usage)
+- [Prefix Options](#prefix-options)
+- [Console](#console)
+- [GraphQL](#graphql)
+- [Using Development Version](#using-development-version)
+- [Relative Cursor Pagination](#relative-cursor-pagination)
+- [Limitations](#limitations)
+- [Additional Resources](#additional-resources)
+
```

Comment:
+++++1 for a table of contents!!!


This shows us more clearly that this project's README is close to the "too big for one file" edge